src/directLine.ts

```diff
@@ -252,6 +261,7 @@ export interface IBotConnection {
     connectionStatus$: BehaviorSubject<ConnectionStatus>,
     activity$: Observable<Activity>,
     end(): void,
+    getConversation(): Conversation,
```

Comment:
The reason it hangs off of IBotConnection is because that is the only reference the WebChat has.